Articles/Setup/Install/Unattended-Install.md

```diff
@@ -0,0 +1,124 @@
+---
+v8-equivalent: "https://github.com/umbraco/UmbracoDocs/blob/main/Getting-Started/Setup/Install/Unattended-Install.md"
+versionFrom: 9.0.0
+verified-against: alpha-4
+state: partial
+updated-links: false
+---
+
+# Unattended Installs
+
+In some cases, you might need to install one or multiple Umbraco instances automatically without having to run through the installation wizard to configure the instance.
+
+You can use the **Unattended installs** feature to allow for quick installation and set up of Umbraco instances on something like Azure Web Apps.
+
+This article will give you the details you need to install Umbraco unattended.
+
+## Get the correct version of Umbraco
+
+Since its beginning **Umbraco version 9** supports this feature.
+
+So it will be enough to get a clean instance from either the [NuGet feed](https://www.nuget.org/packages/UmbracoCms/) or download a zip file directly from [Downloads](https://our.umbraco.com/download).
```

Comment:
I see your point! I would say, it would be nice to add a reference as to where to get a clean Umbraco instance, so the article is complete (we have all the info for attempting unattended install). Maybe something like this?
```suggestion
## Get clean install of Umbraco

In order to get a clean instance of Umbraco either visit the [NuGet feed](https://www.nuget.org/packages/UmbracoCms/) or download a zip file directly from [Downloads](https://our.umbraco.com/download).
```

Reply:
Yes, I like it! 💪